ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/pathbased/common/worldview/accumulator/PathBasedWorldStateUpdateAccumulator.java

```diff
@@ -160,8 +162,6 @@ public void importStateChangesFromSource(
                   });
             });
     storageToClear.addAll(source.storageToClear);
-    storageKeyHashLookup.putAll(source.storageKeyHashLookup);
```

Comment:
why are we removing that ?

Reply:
doing a copy of the slot hash cache could help during //

Reply:
the same if we want remove it I think we need to check with performance impact during // @ahamlat